src/helpers/partnersReducer.ts

```
@@ -44,54 +43,42 @@ export const partnersReducer = (
                             return { ...filter, active: !filter.active }
                         }
                         return filter
                     })
 
-                    // Return a new field object with the updated fields array
                     return { ...field, fields: updatedFields }
                 }
 
-                // Return the field as-is if it doesn't contain the target filter
                 return field
             })
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)